resources/views/user/password/edit.blade.php

```diff
@@ -51,34 +51,7 @@ class="form__text"
                         <label class="form__label form__label--floating" for="current_password">Current Password</label>
                     </p>
                 @endif
-                <p class="form__group">
-                    <input
-                        id="new_password"
-                        class="form__text"
-                        autocomplete="new-password"
-                        minlength="12"
-                        name="new_password"
-                        placeholder=""
-                        required
-                        type="password"
-                        value="{{ old('new_password') }}"
-                    >
-                    <label class="form__label form__label--floating" for="new_password">New Password</label>
-                </p>
-                <p class="form__group">
-                    <input
-                        id="new_password_confirmation"
-                        class="form__text"
-                        autocomplete="new-password"
-                        minlength="12"
-                        name="new_password_confirmation"
-                        placeholder=""
-                        required
-                        type="password"
-                        value="{{ old('new_password') }}"
-                    >
-                    <label class="form__label form__label--floating" for="new_password_confirmation">Repeat Password</label>
-                </p>
+                <livewire:password-strength/>
```

Comment:
```suggestion
                <livewire:password-strength />
```
Leaving out the space just feels wrong for some reason, even though html 4 is long ago. Disregard if you feel the opposite.